src/utilities/constants.js

```diff
@@ -7,5 +7,5 @@ export const MILLISECONDS = {
   THIRTY_MINUTES: 1000 * 60 * 30,
   FIVE_MINUTES: 1000 * 60 * 5,
   ONE_DAY: 1000 * 60 * 60 * 24,
-  ONE: 1000,
+  ONE_SECOND: 1000,
```

Comment:
Renamed this and this needed the change in the only place it was used `TemperatureSyncActions`